docker-compose/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
 
   # WHISK CONTROLLER
   controller:
-    image: whisk/controller:latest
+    image: openwhisk/controller:6a56c45
```

Comment:
Currently I have a jenkins build here https://builds.apache.org/view/OpenWhisk/job/OpenWhisk-DockerHub/
Using jenkinsfile https://github.com/openwhisk/openwhisk/blob/master/tools/jenkins/apache/dockerhub.groovy

Lets add a jenkinsfile for docker-compose testing here: https://github.com/apache/incubator-openwhisk-devtools/blob/master/tools/jenkins/apache/docker-compose.groovy

I think we can merge this PR without the jenkins integration enable yet and create an issue,  if it works locally, then I can create a new Jenkins Job definition that uses the new jenkins file docker-compose.groovy